### ISSUE TITLE
[Electricraft] Induction Motors hissing/sizzling with proper Source limits

### ISSUE THREAD

**Issue by ranger135xp** [ReikaKalseki/Reika_Mods_Issues#1273]
Reika, you must not like reading.

TL:DR I have a tungsten upgraded induction motor accepting current from 2 sources. According to your book, this should work as it is a "limit" not a requirement.
![2016-12-25_20 07 09](https://cloud.githubusercontent.com/assets/15115164/21473541/cc57541a-cadd-11e6-8ad2-ba3c52171021.png)

2 induction generators:
![2016-12-25_20 08 53](https://cloud.githubusercontent.com/assets/15115164/21473550/14df04a8-cade-11e6-99c0-bdf502e5ef70.png)

1 induction motor:
![2016-12-25_20 09 01](https://cloud.githubusercontent.com/assets/15115164/21473552/1bb13544-cade-11e6-86f5-11398ca355f1.png)

Proof it is 4x amplified and has been upgraded with Tungsten (should accept 4 maximum sources according to you)
![2016-12-25_20 09 18](https://cloud.githubusercontent.com/assets/15115164/21473555/343b5ae0-cade-11e6-8b7a-bb26bfefaa3d.png)

It sizzles and hisses, signaling heat gen and power loss.

**Comment by ReikaKalseki** [comment]
8 > 4.

**Comment by ranger135xp** [comment]
You are such a piece of work. You completely set me up to fail. You could've said, like a normal human being, "Oh I made a change to where it looks at how many engines are feeding the ElC network so the 8 engines is greater than the source limit of 4."

I searched far and wide for the answer myself because I dread having to talk to you for any reason. Why you have to be a junior Flowerchild is beyond me. Don't come at me like you're so high and mighty that you can just assume that I'm supposed to know this change was made between v9b and 16d when your change logs don't even reference it. Your RoC handbook has sufficient information but the ElC one is almost useless.

Being a mod author doesn't grant you the right to be a dick.

**Comment by ReikaKalseki** [comment]
The change logs *do* reference it, and it has been reported *ad nauseam* since then, almost invariably - as you did - as a bug. I do not buy for a second you "searched far and wide" when you failed to find half a dozen bug reports, two forum threads, and probably over 40 individual posts on my main thread.

**Comment by ranger135xp** [comment]
Unless you're referencing "ElectriCraft: Network now passes power source info properly", that's all there is in your change logs on https://sites.google.com/site/reikasminecraft/changelogs. Other than that, there's no guidance on what your mods consider as sources; sinks are obvious IMO. Now I understand that it's viewed from the perspective of the ElC network as a whole. For example, in my case, RoC is feeding 8 engines into one ElC network. Previously, it seemed like "sources" meant induction generators feeding into an ElC network. 

I understand that taking the responsibility of being an active mod author for this game is taxing. I've read all your FAQ/EAQ content on your website and the crazy posts from some of these knuckle-draggers. I get that you're most-likely an engineer, but please utilize all the handbooks like you did for RoC. I don't require interactive coloring books and for you to hold my hand while I learn how to play with your mods, but for crying out loud, "ElectriCraft: Network now passes power source info properly" leaves more questions than answers.

I'll agree that Google tends to be a living search engine. I've researched things one day, found nothing, then researched the same query another day to find relevant threads from years past. I posted here on GitHub to allow another person to answer. I believe about two years ago, I posted a few things concerning bugs or whatnot on MC forums and legitimate stuff gets buried by the "ur modz brke meh game!!!" posts.

This is what I get when searching your GitHub: https://github.com/ReikaKalseki/Reika_Mods_Issues/issues?utf8=%E2%9C%93&q=induction.

I'm not claiming to be a GitHub master but I shouldn't have to be a master at something to ask a question or find the answer. Like I said, had the ElC book either say "A source is any engine or machine feeding power to the network", or you say something similar to "each engine outputting power is a source", this would've been done. You might as well replace the ElC handbook with "read my threads and GitHub."

I'm not looking for a fight, but short posts like "working as intended" and "8 > 4" don't contribute to the type of community and resource GitHub is intended to be. The only other mod author to give me some-what of a rough time was Vaskii because he closed an issue without resolution until another user pointed out a flaw in a function call. https://github.com/Vazkii/Botania/issues/1458.

I'm done with this thread. "I do not buy for a second you "searched far and wide" when you failed to find half a dozen bug reports, two forum threads, and probably over 40 individual posts on my main thread." -- this is why you created the handbooks, sir.

TL/DR:
Handbook needs updating. Walking away now.
